src/screens/History/MultiChainHistoryEtherspot.js

```diff
@@ -34,27 +33,67 @@ import { accountHistorySelector } from 'selectors/history';
 import { accountAssetsSelector } from 'selectors/assets';
 import { accountCollectiblesHistorySelector } from 'selectors/collectibles';
 
+// Constants
+import { CHAIN } from 'constants/chainConstants';
+
 // Utils
 import { getHistoryEventsFromTransactions, parseHistoryEventFee } from 'utils/history';
 import { addressesEqual, getAssetsAsList } from 'utils/assets';
+import { useChainsConfig } from 'utils/uiConfig';
 
 // Types
 import { EVENT_TYPE, type Event } from 'models/History';
+import type { Chain } from 'models/Chain';
 
-function HistoryListEtherspot() {
-  const items = useHistoryEvents();
+function MultiChainHistoryEtherspot() {
+  const [tabIndex, setTabIndex] = useState(0);
 
-  return <HistoryList items={items} />;
+  const chainsConfig = useChainsConfig();
+
+  const chains = [
+    CHAIN.ETHEREUM,
+    CHAIN.POLYGON,
+    CHAIN.BINANCE,
+    CHAIN.XDAI,
+  ];
+
+  const items = chains.map((chain) => {
+    const { title } = chainsConfig[chain];
```

Comment:
Full title looks weird in tabs, you probably want to use short one.

```suggestion
    const { titleShort } = chainsConfig[chain];
```

Reply:
Well for me it looks weird as short, especially `BSC` which doesn't say anything for less crypto savvy user 🤔